docs/artifacts/five-clauses-zero-mechanisms.html

```
@@ -0,0 +1,525 @@
+<title>Five Clauses, Zero Mechanisms</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)